datacube_utilities/query.py

```diff
@@ -26,8 +27,9 @@ def create_base_query(aoi, res, output_projection, aoi_crs, dask_chunks):
         "output_crs": output_projection,
         "resolution": resolution,
         "dask_chunks": dask_chunks,
-        "crs": aoi_crs,
+        "crs": "EPSG:3460",
     }
+    logging.info(f"{query}")
```

Comment:
maybe a more meaningful info with timestamp and module name would be better

Reply:
Oops I missed one of the log statements I meant to remove.